android/app/src/main/java/app/organicmaps/widget/menu/NavMenu.java

```diff
@@ -211,8 +216,20 @@ private void updateSpeedView(@NonNull RoutingInfo info)
 
     Pair<String, String> speedAndUnits = StringUtils.nativeFormatSpeedAndUnits(last.getSpeed());
 
+    if (mCurrentSpeedLimit > 0.0)
+    {
+      Pair<String, String> speedLimitAndUnits = StringUtils.nativeFormatSpeedAndUnits(last.getSpeed());
+      mSpeedValue.setText(speedAndUnits.first + " / " + speedLimitAndUnits.first);
```

Comment:
Would this narrow non-breaking space look better? https://www.compart.com/en/unicode/U+202F

Reply:
It can also be used on iOS btw.

Reply:
You mean replace `" / "` with `"\u202F/\u202F"`?

Reply:
Yes, or just copy the necessary space symbol and insert it directly into the string.